EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/EventGenerator.java

```diff
@@ -82,10 +70,10 @@ public void generateEvents(int eventCount, int scaleFactor, EapEventType eventTy
      * @param eventCount number of events to be generated
      * @param scaleFactor time-scale for the replayer to generate the events
      * @param eventType type the generated events should be of
-     * @param attributeSchemas map containing all eventType-attributes and their user-defined, possible value
      * @param eventTimestamps timestamps for the events to be generated with
+     * @param attributeUserInput
      */
-    public void generateEvents(int eventCount, int scaleFactor, EapEventType eventType, Map<TypeTreeNode, String> attributeSchemas, String
+    void generateEvents(int eventCount, int scaleFactor, EapEventType eventType, List<AttributeInput> attributeUserInput, String
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Expected @param tag for 'attributeUserInput'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AJavadocMethod_BP2016)